docs: build.assetsDir only used in non lib mode (#12119)
vitejs/vite: docs/config/build-options.md

- **Type:** `string`
- **Default:** `assets`

> Specify the directory to nest generated assets under (relative to `build.outDir`, only used in non-[Library Mode](/guide/build#library-mode)).

Reviewer: ```suggestion
Specify the directory to nest generated assets under (relative to `build.outDir`. This is not used in [Library Mode](/guide/build#library-mode)).
```
